lib/linter-reek.js

```diff
@@ -19,6 +16,41 @@ const loadDeps = () => {
   }
 };
 
+const parseReekSyntaxError = (error, filePath) => {
+  const exceptionMessage = /Exception message:\s*(.*)/g.exec(error);
+  return [{
+    severity: 'error',
+    excerpt: exceptionMessage ? `linter-reek: ${exceptionMessage[1]}` : 'linter-reek: unexpected error',
+    location: {
+      file: filePath,
+      // first line of the file
+      position: [[0, 0], [0, Infinity]],
+    },
+  }];
+};
+
+const reekSmellToLinter = (smell, file, editor) => smell.lines.map(line => ({
+  url: smell.documentation_link,
+  description: () => RuleHelpers.getRuleMarkDown(smell.smell_type, smell.documentation_link),
+  excerpt: `${smell.smell_type}: ${smell.context} ${smell.message}`,
+  severity: 'warning',
+  location: {
+    file,
+    position: helpers.generateRange(editor, line - 1),
+  },
+}));
+
+const parseFromStd = (stdout) => {
```

Comment:
```suggestion
const parseJSON = (stdout) => {
```
Or some other name that you feel is appropriate, I don't see a reason to include where the data is coming from in the name.